numbers-csharp/ConvertString2Int/ConvertString2IntTests/ConvertString2IntTests.cs

```diff
@@ -0,0 +1,58 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace ConvertString2IntTests
+{
+    [TestClass]
+    public class ConvertString2IntTests
```

Comment:
As I've mentioned in the article, try to create some tests that don't cover just the "happy path". It's important to test for exceptions as well. You'll see what I mean if you read the article review first. Cover empty strings and alphanumeric strings as well. It's a small article and it's easy to cover these edge cases.